src/app/route/route-list/route-list.component.css

```diff
@@ -77,6 +77,27 @@
   margin-right: 10px;
 }
 
+.mobile-grid {
```

Comment:
after adding mobile-grid and desktop-grid classes is list-group class (line 2) still necessary?